ckan/lib/jinja_extensions.py

```diff
@@ -323,12 +324,33 @@ class ResourceExtension(BaseExtension):
 
     @classmethod
     def _call(cls, args, kwargs):
+        log.warn(
+            '`resource` tag is deprecated. '
+            'Use `assets`'
+            '<https://docs.ckan.org/en/latest/theming/webassets.html> instead')
```

Comment:
maybe not worth spamming the logs with this one. I think marking deprecated in release notes is good enough

Reply:
Yes, probably so.